### ISSUE TITLE
Cleaning old unused entires after config edit

### ISSUE THREAD

**Issue by phenomeus** [sahilchaddha/homebridge-yeelight-platform#15]
**Describe the bug**
I still the removed entries like "calm" "party" "police" etc after I cleaned the config and restarted homebridge service

**Expected behavior**
the entries should go away :) please

i also started more or less the same discussion for another plugin but also as a general conversation about best practice to handle old unused entries after plugin deletion or reconfiguring the config file

https://github.com/nfarina/homebridge/issues/2120

**Comment by iChUdY** [comment]
My "waste" room is still filled with other unused scenes. Is there really no way to remove them?

**Comment by iwonder-1618** [comment]
@sahilchaddha will be releasing a new version for the plugin, which should solve the issue. But if you are looking for a quicker hacky solution. You might want to remove `.homebridge/accessories` and `.homebridge/persist` folder.

**Comment by phenomeus** [comment]
this suggestion is not really a quick solution. its a step back because I have to config every entry over again. that is a lot and is not really helpful. 

so i will wait for a plugin update. thanks in advance :)

**Comment by iChUdY** [comment]
Nah.. in no case can I remove .homebridge/accessories and .homebridge/persist folder.

**Comment by Lon1st** [comment]
> @sahilchaddha will be releasing a new version for the plugin, which should solve the issue. But if you are looking for a quicker hacky solution. You might want to remove `.homebridge/accessories` and `.homebridge/persist` folder.

Is there an ETA for this release?

**Comment by phenomeus** [comment]
> @sahilchaddha will be releasing a new version for the plugin, which should solve the issue. But if you are looking for a quicker hacky solution. You might want to remove `.homebridge/accessories` and `.homebridge/persist` folder.

when sir? :D

**Comment by ZABAYA** [comment]
If you download the app Controller for Homekit you can make backups for your accessoires and scenes. If you're tinkering with your home setting as much as i do it's worth the 8 euro for the premium version.

**Comment by phenomeus** [comment]
Backups are good but only can do a full restore. Homebridge now got the -R handle. This is already helping

**Comment by sahilchaddha** [comment]
On it.

**Comment by phenomeus** [comment]
ETA :D ?
